UnityAuth/src/main/java/io/unityfoundation/auth/AuthController.java

```diff
@@ -105,14 +105,19 @@ private List<String> checkUserPermission(User user, Tenant tenant, List<String>
 
   private HttpResponse<HasPermissionResponse> createHasPermissionResponse(boolean hasPermission,
                                                                           String userEmail,
-                                                                          String message, List<String> permissions) {
-    return HttpResponse.ok(new HasPermissionResponse(hasPermission, userEmail, message, permissions));
+                                                                          String firstName,
+                                                                          String lastName,
+                                                                          String message,
+                                                                          List<String> permissions) {
+    return HttpResponse.ok(new HasPermissionResponse(hasPermission, userEmail, firstName, lastName, message, permissions));
   }
 
   @Serdeable
   public record HasPermissionResponse(
       boolean hasPermission,
       @Nullable String userEmail,
+      @Nullable String firstName,
+      @Nullable String lastName,
```

Comment:
We actually don't need the firstName and lastName here but rather on the `AuthenticationResponse`

Reply:
Yes, that's what I was wondering. I put it here as we are already pulling information from the user table in the hasPermission call to get permissions. Should we instead remove the names from the hasPermission call and only include it in the AuthenticationResponse?

Reply:
Yeah I think that is preferable